settings.d/pulpcore.yml.example

```diff
@@ -1,8 +1,22 @@
 ---
 :enabled: true
-#:pulp_url: https://localhost/
-#:content_app_url: https://localhost:24816/
+
+# The URL serving the pulpcore-api without any path
+#:pulp_url: https://localhost:24817/
+# Only set this if the container registry is present. This points to pulpcore-api.
+# Can also be set to true in which case it'll be derived from pulp_url
+#:container_registry_url: https://localhost:24817/pulpcore_registry
```

Comment:
I think the part about pulpcore-api is incorrect. The registry endpoint is present on the content app.

Can you have the registry end point running at a different `host:port` combo that the content app? @jlsherrill

Reply:
Are you certain? I based this on https://github.com/theforeman/puppet-pulpcore/blob/7a5543b1a47de8d76a1cad1fb76e12db7bbf8e45/manifests/plugin/container.pp#L9-L21.

Now that look I see we also have an Ansible Galaxy endpoint:
https://github.com/theforeman/puppet-pulpcore/blob/7a5543b1a47de8d76a1cad1fb76e12db7bbf8e45/manifests/plugin/ansible.pp#L6-L16

Perhaps that should be exposed as well so Katello doesn't have to guess. @jlsherrill is that used anywhere within Katello?

Reply:
This https://github.com/Katello/katello/pull/9450/files#diff-bda566f2c5501c43867744c439cd22a7686af2940052749f3414e7b10c51308cR29-R30 constructs it based on the content app URL. I guess I don't right know which is which right at the moment. Save us Obi-Wan